README.md

```diff
@@ -42,10 +41,71 @@ Turn Ton logging to stderr. Default to false.
 ### `loglevel`
 Defines the logging levels. Default to info. Valid values are: debug, info, warn, error
 
+### Notes:
+This library generates URL Search Parameters (if required) at runtime for RPCs annotated as GET requests. For example, given the following proto definition
+
+```proto
```

Comment:
protos in the integration test already covers this

Reply:
So maybe we can point out in the notes to look for the integration test example?

Reply:
I think so. just point a link to the integration test service.proto will do the work.

Reply:
Understood. Thanks for the feedback 🙇🏼

Reply:
Completed 🙏🏼